Source/Applications/SystemCenter/Model/Configuration/ConfigurationLoader.cs

```diff
@@ -116,8 +116,8 @@ private IEnumerable<string[]> LoadXDASettings(AdoDataConnection connection)
 
         private IEnumerable<string[]> LoadSystemCenterSettings(AdoDataConnection connection)
         {
-            TableOperations<SystemCenter.Model.Setting> settingTable = new TableOperations<SystemCenter.Model.Setting>(connection);
-            List<SystemCenter.Model.Setting> settingList = settingTable.QueryRecords().ToList();
+            TableOperations<Setting> settingTable = new TableOperations<Setting>(connection);
```

Comment:
are you sure that will work? could be referencing the wrong `Settings` model now